packages/example/src/pages/components/ExpressiveList.mdx

```diff
@@ -0,0 +1,135 @@
+</ExpressiveList>
+</div>
+
+<Title>With pictograms</Title>
+
+<div className="expressive-content-list-group">
+<ExpressiveList 
+  title="Short Title"
+  background={true}
+  pictogram={<Connect aria-label="Add" className="my-custom-class" />}
+>
+Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate. Orci ac auctor augue mauris augue neque. Nullam ac tortor vitae purus faucibus ornare suspendisse sed. 
+</ExpressiveList>
+<ExpressiveList 
+  title="Short Title"
+  background={true}
+  pictogram={<Connect aria-label="Add" className="my-custom-class" />}
+>
+Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate. Orci ac auctor augue mauris augue neque. Nullam ac tortor vitae purus faucibus ornare suspendisse sed. 
+</ExpressiveList>
+<ExpressiveList 
+  title="Short Title"
+  background={true}
+  pictogram={<Connect aria-label="Add" className="my-custom-class" />}
+>
+Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate. Orci ac auctor augue mauris augue neque. Nullam ac tortor vitae purus faucibus ornare suspendisse sed. 
+</ExpressiveList>
+</div>
+
+## Code
+
+<Title>Content only</Title>
+
+```mdx path=components/ExpressiveList/ExpressiveList.js src=https://github.com/carbon-design-system/gatsby-theme-carbon/tree/main/packages/gatsby-theme-carbon/src/components/ExpressiveList
+<div className="expressive-content-list-group">
+  <ExpressiveList 
+    title="Short Title"
+    background="true">
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+
+  <ExpressiveList 
+    title="Short Title"
+    titleType="expressive-03"
+    background="true">
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+    
+  <ExpressiveList 
+    title="Short Title"
+    titleType="expressive-02"
+    background="true">
+  Lorem ipsum tempor incididunt ut labore et dolore magna aliqua. Quam lacus suspendisse faucibus interdum posuere lorem. At quis risus sed lorem ipsum vulputate.
+  </ExpressiveList>
+</div>
+```
+
+<Title>With pictograms</Title>
+
+```mdx path=components/ExpressiveList/ExpressiveList.js src=https://github.com/carbon-design-system/gatsby-theme-carbon/tree/main/packages/gatsby-theme-carbon/src/components/ExpressiveList
+<div className="expressive-content-list-group">
```

Comment:
I think I would add this to the example code, otherwise I could easily see someone forget to add it and wonder why it wasn't showing up 😄 

`import { Connect } from '@carbon/pictograms-react';`